app/views/contexts/section_summary_context.py

```diff
@@ -136,47 +160,52 @@ def _list_summary_element(self, summary, current_location, section) -> Mapping:
             primary_person_edit_block_id = primary_person_block["add_or_edit_block"][
                 "id"
             ]
+            edit_block_id = primary_person_block["add_or_edit_block"]["id"]
 
         add_link = self._add_link(
-            summary, current_location, section, list_collector_block
-        )
-
-        list_context = ListContext(
-            self._language,
-            self._schema,
-            self._answer_store,
-            self._list_store,
-            self._progress_store,
-            self._metadata,
+            summary, section, list_collector_block, list_collector_blocks_on_path
         )
 
         rendered_summary = self._placeholder_renderer.render(
             summary, current_location.list_item_id
         )
 
+        list_summary_context = {}
+
+        if list_collector_block:
```

Comment:
```suggestion
```

this needs to be not conditional 👍

Reply:
This conditional has been removed with changes from previous comment.